channels/in.m3u

```diff
@@ -11,6 +11,10 @@ http://103.199.161.254/Content/7smusic/Live/Channel(7smusic)/index.m3u8
 https://d2q8p4pe5spbak.cloudfront.net/bpk-tv/9XM/9XM.isml/index.m3u8
 #EXTINF:-1 tvg-id="ANEWSJATTTVBhangraFlava.in" tvg-name="A NEWSJATT TV- Bhangra Flava" tvg-country="IN" tvg-language="Malayalam" tvg-logo="" group-title="Music",A NEWSJATT TV- Bhangra Flava (720p)
 http://newsjatt.camdvr.org:1935/newsjatt/myStream/playlist.m3u8
+#EXTINF:-1 tvg-id="ANEWSdonsTVBhangraFlava.in" tvg-name="A NEWSDONs TV- SWAG SADDA VAKRAA" tvg-country="IN" tvg-language="punjabi" tvg-logo="" group-title="Music",A NEWSDONs TV- SWAG SADDA VAKRAA (720p)
+http://tiny.cc/jattvlink
+#EXTINF:-1 tvg-id="ANEWSdonsTVBhangraFlava.in" tvg-name="A NEWSDONs TV- SWAG SADDA VAKRAA" tvg-country="IN" tvg-language="punjabi" tvg-logo="" group-title="Music",A NEWSDONs TV- SWAG SADDA VAKRAA (LINK 2)
+https://video-weaver.maa01.hls.ttvnw.net/v1/playlist/CrgFG968dAy-iSQltxXB3lsgBtnO1zu2KQrfK3cGuHfabQAMGRPotoKhio0ElzuriYxH0USwnm9ZfDXMr1lt9gq1V81MKfrijHIaZMGyH_9XqgGV-7E9pG2Q8siVbnmSDuvsTCo4QEvV9HzTaShHsuHET4SENXa-QY4dMBUhO_5lqo37ktzW5ICJRIcqDRDYF_bc5fKd3Si_lGcti9n_oOq4FEoifSjbZX5ckUMwYUKrfTSLrYQKyjaTm107zNZnSDLEI2JTCmwnwtDP0LoakFKBCv5g0De2dtY85EWIuvIAYp0y6tBvkXgMd7G4-Hm9U8WdLA86HdVLLwGBY0snt2DZW-Er8e867ueUBwJeCpIS2OFhhuNNWpA_i6Z8KSGxfSjWQGm_VbqdQFjOs3akUkuv4lcQ34Mr3amhWZBvA3pD_g6J_P6Gt8EHqR_irbikzqdkvob9aQEll7o1bOtghCiFty5pmEjlbGY-0sdYcG1uxhXlpUGD6kO1tl6LA7Wb83C30L6OxK1rqavkINZDfh6xwkcMO547cavpXtRR9iLMQdP6vjR8sSr3N49TTsG1dx9HaMeLAH2nVPFkLJZnmdqYZe-4hvV_e-4sDie-uP_XXNv9kakEad68eRx_dBFB9U-1IG2GldNJcsXEZMN1GFeKSWxSrUDUs_FIlCksAxXwgAakii-kwgHkf7c2ZHdzI9Z6ZWVpceKBO7ORpr3R_N2hkcQDg1zbPPI53YCv7e3Jn8GUYfRJ8zqChQbJJQlGGuRHGvcxjsW8y-14NgWwXPYYOHaHMyebB_LZ1R7vJ-TRk_GJRvPqR3N1h8OqYALx5JCmtTvHdq7M7Ci8QuUnXkHFqiXuq9hPqcXFD3Eu0zsFWjhbE9MGIRNBKL9nyTcWdpL37CKFvDeifvSbGSxOuVfjvOBntX6j6kPsEhD_ZvHY_X5xytAnkNDRm6YkGgySVeNRdfNR6OaqZJw.m3u8
```

Comment:
This is not a valid link.

Reply:
2nd one doesn't work ERROR 404 NOT FOUND

Reply:
Pasting a Twitch playlist link is not useful. You can use the Streamlink query hosts if you want to use a Twitch link.